src/app/library/sagas.ts

```diff
@@ -0,0 +1,32 @@
+import { SagaIterator } from 'redux-saga';
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { client } from '../../graphql/api';
+
+import { ADD_PLAYLIST, LOAD_LIBRARY } from './actions';
+
+export function* loadLibrarySaga(action: ReturnType<typeof LOAD_LIBRARY.TRIGGER>): SagaIterator {
+    try {
+        const result = yield call(client.getLibrary, action.payload);
```

Comment:
You should add .STARTED, when you setting isFetching to true, also you should display ActivityIndicator in the screen

Reply:
Made it in TRIGGER action